src/briefcase/commands/publish.py

```diff
@@ -33,7 +33,9 @@ def publish_app(self, app: BaseConfig, channel: str, **options):
         :param app: The application to publish
         :param channel: The publication channel to use
         """
-        self.logger.info(f"TODO: Publish {app.app_name} to {channel}")
+        self.logger.info(
+            f"TODO: Publish {app.app_name} to {channel}"
+        )  # pragma: no cover
```

Comment:
This is effectively a `raise NotImplementedError()`, but with nicer optics if it's invoked.